pkg/api/steve/clusters/shell.go

```diff
@@ -102,6 +102,17 @@ func (s *shell) createPod() *v1.Pod {
 		Spec: v1.PodSpec{
 			TerminationGracePeriodSeconds: new(int64),
 			RestartPolicy:                 v1.RestartPolicyNever,
+			NodeSelector: map[string]string{
+				"kubernetes.io/os": "linux",
+			},
+			Tolerations: []v1.Toleration{
+				{
+					Key:      "cattle.io/os",
+					Operator: "Equals",
```

Comment:
Does `Equals` work ? The [docs only show](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) `Equal` or `Exists`.